source/air/src/ast.rs

```diff
@@ -98,14 +98,16 @@ pub enum Quant {
 pub type Trigger = Exprs;
 pub type Triggers = Arc<Vec<Trigger>>;
 
+pub type Qid = Option<String>;
```

Comment:
What about `Qid = Option<Ident>`? (Where `Ident` is `Arc<String>`)
I think we do the same for debugger tags, and considering you're cloning `Qid`s from time to time, I think this would be a bit more consistent (and possibly marginally faster). (If at some point we decide we need a string interner/allocator, it's easier to find all the AST Idents this way, too.)